docs/manpages/pdns_control.1.md

```diff
@@ -72,10 +74,10 @@ ping, rping
 purge [*RECORD*]
 :    Purge entries from the packet cache. If *RECORD* ends with a dollar ($)
      all entries that end with that name are removed. If no record is specified
-     the entire cache is purged.
+     the entire packet cache is purged.
```

Comment:
If I read `DLPurgeHandler` in pdns/dynhandler.cc correctly (https://github.com/PowerDNS/pdns/blob/master/pdns/dynhandler.cc#L122), we don't touch the querycache on a purge command.